.github/workflows/unittests.yml

```diff
@@ -126,8 +126,14 @@ jobs:
         run: |
           curl -L https://github.com/bazelbuild/bazelisk/releases/download/v1.3.0/bazelisk-linux-amd64 > bazelisk
           chmod +x bazelisk
+      - name: Force Python 3 for everything we do
+        run: |
+          ln -sf /usr/bin/python3.6 /usr/bin/python
+          ln -sf /usr/bin/python3.6 /usr/bin/python3
```

Comment:
Hopefully we can soon figure out how to do this properly. Maybe simply uninstalling python2 at the start?

Reply:
Unfortunately this is also how [Tensorflow Addons](https://github.com/tensorflow/addons/blob/master/tools/ci_build/builds/release_linux.sh#L45) solves their Python version problem. I agree there must be a better way to do this, but it would involve messing with `tf_configure` or trying to find the correct environment variable to set.
I think I'd be in favour of just running the tests on macos outside the container with a custom the github provided Python version which should make CI faster as well.

Reply:
Just for reference, I think [this is the code](https://github.com/tensorflow/tensorflow/blob/4f2afc07b26748f80a0de768fac81c7816410b44/configure.py#L203-L269) that determines the Python version used by TensorFlow (and the TensorFlow provided pybind extension).

Reply:
Or maybe we could look into switching to the default `pybind_extension` https://github.com/pybind/pybind11_bazel

Reply:
I think we also tried that for some other bazel command: switching from the TF version to the normal bazel version, but then somehow things got messed up.

The code you linked that determines the Python version suggests setting `PYTHON_LIB_PATH` and `PYTHON_BIN_PATH`, but if this is how TF Addons solves it, then I'm 100% in favor of leaving it like this.